src/Adapter/LocalStorage.php

```diff
@@ -197,6 +197,14 @@ public function getURL($path)
 
         return $this->webUrl.'/'.$path;
     }
+    
+    /**
+     * {@inheritDoc}
+     */
+    public function getExpiringURL($path, $expiresAt) {
+        // Local storage cannot provide expiring URLs. Falling back...
```

Comment:
That seems like a lot of setup for the local env then. Don't forget that the file isn't deleted once the signed URL expires, just the access to it. (So it'd be more accurate to create a link and delete the link at expiry)